libs/feature/editor/src/lib/components/contact-card/contact-card.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core'
+import {
+  Individual,
+  Organization,
+} from '@geonetwork-ui/common/domain/model/record'
+import { MatIconModule } from '@angular/material/icon'
+import { CommonModule } from '@angular/common'
+import { ButtonComponent } from '@geonetwork-ui/ui/inputs'
+
+@Component({
+  selector: 'gn-ui-contact-card',
+  templateUrl: './contact-card.component.html',
+  styleUrls: ['./contact-card.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [CommonModule, MatIconModule, ButtonComponent],
+})
+export class ContactCardComponent {
+  @Input() contact: Individual
+  @Input() organization: Organization
```

Comment:
I'm surprised that this input is needed; the `Invididual` model already contains an organization, shouldn't that be enough?